integration_test/general.py

```diff
@@ -0,0 +1,154 @@
+    return runCommand("export PATH=/opt/apache-maven-3.6.3/bin:$PATH &&"
+           "cd dcm4che/dcm4che-tool/dcm4che-tool-storescu &&"
+           "mvn -ntp exec:java -Dexec.mainClass=org.dcm4che3.tool.storescu.StoreSCU -Dexec.args=-\"c IMPORTADAPTER@"+host+":"+adapter_port+" "+file_path+"\"", "run-store-scu exit with")
+
+# run-find-scu-instance
+def run_find_scu_instance(host, adapter_port):
+    return runCommand("export PATH=/opt/apache-maven-3.6.3/bin:$PATH &&"
+           "cd /workspace/dcm4che/dcm4che-tool/dcm4che-tool-findscu &&"
+           "mvn -ntp  exec:java -Dexec.mainClass=org.dcm4che3.tool.findscu.FindSCU -Dexec.args=\"-c IMPORTADAPTER@"+host+":"+adapter_port+" -L IMAGE -X --out-cat --out-file findscu-instance-result.xml --out-dir ../../../integration_test/\"", "run-find-scu-instance exit with")
+
+# run-find-scu-series
+def run_find_scu_series(host, adapter_port):
+    return runCommand("export PATH=/opt/apache-maven-3.6.3/bin:$PATH &&"
+           "cd /workspace/dcm4che/dcm4che-tool/dcm4che-tool-findscu &&"
+           "mvn -ntp  exec:java -Dexec.mainClass=org.dcm4che3.tool.findscu.FindSCU -Dexec.args=\"-c IMPORTADAPTER@"+host+":"+adapter_port+" -L SERIES -X --out-cat --out-file findscu-series-result.xml --out-dir ../../../integration_test/\"", "run-find-scu-series exit with")
+
+# run-find-scu-study
+def run_find_scu_study(host, adapter_port):
+    return runCommand("export PATH=/opt/apache-maven-3.6.3/bin:$PATH &&"
+           "cd /workspace/dcm4che/dcm4che-tool/dcm4che-tool-findscu &&"
+           "mvn -ntp  exec:java -Dexec.mainClass=org.dcm4che3.tool.findscu.FindSCU -Dexec.args=\"-c IMPORTADAPTER@"+host+":"+adapter_port+" -L STUDY -X --out-cat --out-file findscu-study-result.xml --out-dir ../../../integration_test/\"", "run-find-scu-study exit with")
+
+# run-move-scu
+def run_move_scu(host, adapter_port):
+    return runCommand("export PATH=/opt/apache-maven-3.6.3/bin:$PATH &&"
+           "cd /workspace/dcm4che/dcm4che-tool/dcm4che-tool-movescu &&"
+           "mvn -ntp  exec:java -Dexec.mainClass=org.dcm4che3.tool.movescu.MoveSCU -Dexec.args=\"-c IMPORTADAPTER@"+host+":"+adapter_port+" --dest STORESCP\"", "run-move-scu exit with")
+
+# run-commitment-scu
+def run_commitment_scu(host, adapter_port, com_scu_port, file_path):
+    return runCommand("export PATH=/opt/apache-maven-3.6.3/bin:$PATH &&"
+           "cd /workspace/dcm4che/dcm4che-tool/dcm4che-tool-stgcmtscu &&"
+           "mvn -ntp  exec:java -Dexec.mainClass=org.dcm4che3.tool.stgcmtscu.StgCmtSCU -Dexec.args=\"-c IMPORTADAPTER@"+host+":"+adapter_port+" -b STGCMTSCU:"+com_scu_port+" --explicit-vr --directory /workspace/integration_test/commitment_result "+file_path+"\"", "run-commitment-scu exit with")
+
+# close-store-scp
+def close_store_scp(store_scp_run_step, close_store_scp_port):
+    return runCommand("apt-get install -y netcat && nc -z "+store_scp_run_step+" "+close_store_scp_port, "close-store-scp exit with")
```

Comment:
This is no longer necessary (see my below comment on netcat) so this command could be removed from all tests.